src/panels/README.md

```diff
@@ -1,27 +1,48 @@
+
+![Panel hero](/docs/assets/images/panels-hero.png)
+
+The component is created using three components that work together to interchangably display different content:
 
 -   `<vscode-panels>`: The top level container element.
 -   `<vscode-panel-tab>`: Renders the panel tab that will be associated with a panel view.
 -   `<vscode-panel-view>`: The container element that will hold content associated with a given tab.
 
-## Panels Attributes
+## Usage
+
+| ❌ Don't                                                                                      | ✅ Do                                                                               |
+| --------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- |
+| ![Tab panel with too many tabs](/docs/assets/images/panels-dont-1.png)                        | ![Tab panel with several clearly labeled tabs](/docs/assets/images/panels-do-1.png) |
+| Don't use more than six panel tabs to organize your extensions views to reduce load on users. | Structure content into as few tabs as possible.                                     |
+
+| ❌ Don't                                                                     | ✅ Do                                                           |
+| ---------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| ![Tab panel with random tab ordering](/docs/assets/images/panels-dont-2.png) | ![Image placeholder](/docs/assets/images/panels-do-2.png)       |
+| Don't use a random tab order.                                                | Tabs with related content should be grouped next to each other. |
+
+| ❌ Don't                                                                                           | ✅ Do                                                                                                                                                                                               |
+| -------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Tab featuring a very long label with multiple categories](/docs/assets/images/panels-dont-3.png) | ![Image placeholder](/docs/assets/images/panels-do-3.png)                                                                                                                                           |
+| Don't use too many words in a tab label.                                                           | Use clear, concise titles to ensure your extensions users know what to expect in each tab. Consider splitting content into multiple tabs if one panel contains many distinct categories of content. |
```

Comment:
Found a spot where the image sizes are really different.

<img width="932" alt="Screen Shot 2022-02-04 at 11 26 42 AM" src="https://user-images.githubusercontent.com/39639992/152590912-dc9cb5f6-d430-48c3-a8aa-a8c45ed39826.png">

Reply:
Switching to use an `img` tag in this particular spot might be adventitious.

Reply:
I've been trying out using `img` tags for these main do/don't images. One issue I ran into is how hardcoding the width leads to some readability issues on smaller viewports.

I've adjusted the text to help make these more even for now—will add an issue to find a better solution that works well on any screen. I'm sure there are a number of options here.